src/libcore/option.rs

```diff
@@ -689,18 +691,23 @@ impl<T> Option<T> {
         }
     }
 
-    /// Inserts `v` into the option if it is None, then
-    /// returns a mutable reference to the contained value
+    /// Inserts `v` into the option if it is `None`, then
+    /// returns a mutable reference to the contained value.
     ///
     /// # Examples
     ///
     /// ```
     /// #![feature(option_entry)]
     ///
     /// let mut x = None;
-    /// let y: &mut u32 = x.get_mut_or_insert(5);
-    /// assert_eq!(y, &5);
-    /// *y = 7;
+    ///
+    /// {
+    ///   let y: &mut u32 = x.get_mut_or_insert(5);
```

Comment:
all this code should be indented another two spaces; four is the standard style.